ChainRouter.php

```diff
@@ -306,4 +306,25 @@ public function getRouteCollection()
 
         return $this->routeCollection;
     }
+    
+    /**
+     * Identify if any routers have been added
+     * 
+     * @return bool
+     */
+    public function hasRouters()
+    {
+        return !empty($this->routers);
```

Comment:
well, we don't need PHP to count() the zval, we just need to see if it has data in it - this is why i use empty() for things when I don't want PHP to calculate the count()